packages/manager/modules/anthos/src/anthos.controller.js

```diff
@@ -0,0 +1,11 @@
+export default class AnthosTenantsCtrl {
+  /* @ngInject */
+  constructor(AnthosTenantsService) {
+    this.AnthosTenantsService = AnthosTenantsService;
+    this.versionPrefix = 'APM';
```

Comment:
Please move this to constants

Reply:
I used it only here.
Do we will used it into another place ?

Reply:
ok...